force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -285,11 +364,22 @@ public with sharing class FlowControlLoopsCollections {
      * Example: countMoney() should return 48.02
      * @return The total value in dollars, stopped at the point when it exceeds 40 dollars.
      */
-    public static Decimal countMoney() {
-        // The list of money in the wallet, represented in dollars and cents
-        List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
 
-        return null; // Replace null with the variable you used to store the result
+    public static Decimal countMoney()
+    {
+        Decimal totalAmount = 0; // Initialize variable
+        List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; // DO NOT CHANGE (PROVIDED)
+
+        for (Decimal change : moneyInWallet) // For loop interates over existing list
+        {
+            totalAmount += change;
+
+            if (totalAmount > 40) // If sum gets over 40, stop the loop
```

Comment:
Try using a variable for the 40 so its not a magic number https://en.wikipedia.org/wiki/Magic_number_(programming)